test/helpers/kubectl.go

```diff
@@ -210,6 +211,22 @@ func (kub *Kubectl) CiliumEndpointsList(pod string) *CmdRes {
 	return kub.CiliumExec(pod, "cilium endpoint list -o json")
 }
 
+//CiliumEndpointsIDs returns a mapping with of a pod name to it is
+//corresponding endpoint ID.
+func (kub *Kubectl) CiliumEndpointsIDs(pod string) map[string]string {
+	filter := `{range [*]}{@.container-name}{"="}{@.id}{"\n"}{end}`
+	return kub.CiliumExec(pod, fmt.Sprintf(
+		"cilium endpoint list -o jsonpath='%s'", filter)).KVOutput()
+}
+
+//CiliumEndpointsIdentityIDs returns a mapping with of a pod name to it's
```

Comment:
Incorrect grammar: "returns a mapping of a pod name to its corresponding endpoint's security identity"